src/services/feed_manager.py

```
@@ -143,9 +147,11 @@
                     feed = Feed(
                         title=feed_data.title,
                         url=feed_data.url,
-                        genre=getattr(feed_data, 'category', 'Other'),
-                        description=getattr(feed_data, 'description', ''),
-                        deleted_at=datetime.fromisoformat(getattr(feed_data, 'deletedAt', datetime.now().isoformat()))
+                        genre=getattr(feed_data, "category", "Other"),
+                        description=getattr(feed_data, "description", ""),
+                        deleted_at=datetime.fromisoformat(
+                            getattr(feed_data, "deletedAt", datetime.now().isoformat())
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)